Jorge-Taylor.txt

```diff
@@ -0,0 +1,192 @@
+https://codepen.io/JorgeT/pen/vYYwgXN
+
+1. What is Semantic HTML? 
+
+    Semantic HTMl gives our code meaning, so we/computer know what's going on by the tags.
+
+    2. What is HTML used for? 
+
+    HTML is used to make websites. It is the foundation of a website.
+
+    3. What is an attribute and where do we put it? 
+
+    An "attribute" is the element modifier in the start tag. or i should say its something added to the start tag so the computer knows the characteristics of the element.
+
+    4. What is the h1 tag used for? How many times should I use it on a page?
+
+    The h1 tag is similar to "Title" tag. 
+    It is used to mark a header or header1 so it will be the largest of the h tags as its importance to be seen is 1st.
+    It can be used as many times as you would like but my personal opinion is it should only be used once in your code.
+
+    5. Name two tags that have required attributes
+          <a "href"=""></a>  <img 'src=""' 'alt=""'>
+
+    6. What do we put in the head of our HTML document?
+       <title></title> we put our "title" (the title of our document). 
+       But prior to that I put <!DOCTYPE html> to let the browser know what version of HTML the doc is writin in.
+ 
+    7. What is an id? 
+ 	Id is used for anchoring not styling .its unique so can only be used once per element.
+
```

Comment:
Id can only be used once PER HTML PAGE so be careful with it. _always try and use a class selector first. But yeah, ids are useful for anchoring, good job!